cmd/service/main.go

```diff
@@ -28,8 +31,47 @@ func startSignalHandler(c context.Context) (context.Context, context.CancelFunc)
 	return ctx, cancel
 }
 
+func logApplicationInfo() {
+	var (
+		version    string = "unknown"
+		revision   string = "unknown"
+		commitTime string = "unknown"
+		dirty      bool   = false
+	)
+	info, ok := debug.ReadBuildInfo()
+	if !ok {
+		return
+	}
+	// version is set to "(devel)" when building from a git checkout
+	if info.Main.Version != "" {
+		version = info.Main.Version
+	}
+
+	for _, s := range info.Settings {
+		if s.Value == "" {
+			continue
+		}
+		switch s.Key {
+		case "vcs.modified":
+			dirty = s.Value == "true"
+		case "vcs.revision":
+			revision = s.Value
+		case "vcs.time":
+			commitTime = s.Value
+		}
+	}
+	slog.Info(
+		"idmsvc-backend",
+		slog.String("Version", version),
+		slog.String("Commit", revision),
+		slog.String("CommitTime", commitTime),
+		slog.Bool("Dirty", dirty),
+	)
+}
+
 func main() {
 	wg := &sync.WaitGroup{}
+	logApplicationInfo()
```

Comment:
I suggest to call this after `logger.InitLogger`; rationalization for getting this message into kibana once we are integrated with clowdWatch.

Reply:
I want to log the info before loading configuration, because `config.Get` can fail with a panic. Isn't Kabana using the standard stream output?

Reply:
my understanding is that it is collected by clowdwatch; envision and content-sources has integration with clowdwatch; asking quickly to be 100% sure.

Reply:
ok! confirmed it is necessary clowdwatch to see the logs in kibana.

Reply:
BTW I didn't realize about the panic, thanks for the heads up!

Reply:
After reviewing the code for the `config.Get`, resolving this conversation; at least we have an entry when the process start in the openshift log if something fails on that operation.